pdal/compression/LazPerfVlrCompression.cpp

```diff
@@ -317,7 +317,7 @@ class LazPerfVlrDecompressorImpl
         if (chunk == m_chunks.end() || nextChunk == m_chunks.end())
             return false;
 
-        m_chunkPointsTotal = nextChunk->count - chunk->count;
+        m_chunkPointsTotal = (int)(nextChunk->count - chunk->count);
```

Comment:
shouldn't this be an explicitly sized type like `int32_t`? I suppose that requires much more refactoring for little value though.

Reply:
`m_chunkPointsTotal` is an int. But I don't think this matters anyway. On any system we're targeting an int is at least 32 bits.